modules/insights-operator-about.adoc

```diff
@@ -14,7 +14,5 @@ The Insights Operator does not collect identifying information, such as user nam
 Red Hat uses all connected cluster information to:
 
 * Proactively identify potential cluster issues and provide a solution and preventive actions in {cloud-redhat-com}
-* Improve {product-title}
+* Improve {product-title} by providing aggregated and critical information to {product-title} product and support teams.
```

Comment:
I think having the full {product-title} listed in one sentence twice sees wordy. My suggestion:

Provide aggregated and critical information to {product-title} product and support teams.

Reply:
@sfortner-RH thank you for your review!

Thanks for highlighting this, I have now removed the second instance of the product name from the sentence. I've kept the "Improve" part of the sentence in because I think that it's important to highlight that is why we are providing the information.